### PR TITLE
Anagram of String in python

### PR DESCRIPTION
**Pull request by AdityaJ7**
### Have you read the [Contributing Guidelines on Pull Requests]

Yes

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #860

### CODE REVIEW

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM 👍

File: Python/cp/anagram_checker.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        #Sort both the strings and check their equality
+        first_string = sorted(first_string.lower())
+        second_string = sorted(second_string.lower())
+        
+        if first_string == second_string:
+            print("These two words are anagrams")
+        else:
+            print("These two words are not anagrams")
+        
+
```
Comment (Kajol-Kumari) [inline Python/cp/anagram_checker.py]: Please remove this extra blank line here.

File: Python/cp/anagram_checker.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#Sample 1
+#Inputs: "silent" and "listen"
+
+#Output: These two words are anagrams
+
+#Sample 2
+#Inputs: "july" and "june"
+
+#Output: These two words are  not anagrams
```
Comment (Kajol-Kumari) [inline Python/cp/anagram_checker.py]: please add one extra blank line at the end of the file.

**Review by Kajol-Kumari — APPROVED** [review]
@AdityaJ7 MInor changes required. Functionality is working fine, code is well organised and LGTM :)

**Comment by AdityaJ7** [comment]
> @AdityaJ7 MInor changes required. Functionality is working fine, code is well organised and LGTM :)

Made the requested changes

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Pull request merged**